skins/oasis/js/WikiaFooter.js

```diff
@@ -30,7 +33,7 @@ var WikiaFooterApp = {
 	},
 	addScrollEvent: function() {
 		'use strict';
-		WikiaFooterApp.windowObj.off( 'scroll.FooterAp' ); // GlobalNotifications could be re-binding this event.
+		WikiaFooterApp.windowObj.off( 'scroll.FooterAp' );
 		WikiaFooterApp.windowObj.on( 'scroll.FooterAp', WikiaFooterApp.resolvePosition ).triggerHandler( 'scroll' );
```

Comment:
Let's create a follow-up ticket to have WikiaFooter use the nifty new onScroll module (if one hasn't been created already).